app/models/katello/sync_plan.rb

```diff
@@ -174,5 +174,25 @@ def rec_logic_changed?
     def cron_status_mismatch?
       self.interval != CUSTOM_CRON && !(self.cron_expression.nil? || self.cron_expression.eql?(''))
     end
+
+    def validate_and_update_products(force_update: false)
+      sync_plan_products = Product.where(id: self.product_ids).select { |p| p.enabled? }
+      if sync_plan_products.length == self.product_ids.length
+        Rails.logger.info "Successfully validated presence of enabled sync plan products!"
```

Comment:
Do we need this to be logged? This will be the case nearly 100% of the time.

Reply:
I was not sure if we should or should not keep it..The more logging the better but I guess we can remove it since it's the happy path and we don't need to log that. 👍🏼

Reply:
Pushed a commit to remove logging in happy path.